tests/tests.py

```diff
@@ -22,3 +29,110 @@ def collect(name, doc):
         generate_csv(f)
         filepaths = export(collector, f.name)
         print(filepaths)
+
+
+suitcase = csv
+
+
+class TestRheology(unittest.TestCase):
+
+    def setUp(self):
+        """Generate test files and headers"""
+        csvpath = 'test.csv'
+        self.header = [('start', {'time': 1534432676.1674924, 'uid': '4becc62f-8f0b-477f-a772-f1c66d6ca70b'}), (
```

Comment:
Is it ok to have hard-coded header here, or better to generate header from RE?

Reply:
Better to generate one, I think, so that it is guaranteed to be realistic and automatically updates when bluesky output evolves